spec/jobs/api_request_job_spec.rb

```diff
@@ -99,8 +100,29 @@
       body: "",
       path: "/api/v1/bar",
       method: "POST",
-    }, {}, 500, Time.zone.now)
+    }, {}, 500, Time.zone.now, nil)
 
     expect(ApiRequest.find_by(request_path: "/api/v1/bar").request_body).to be_nil
   end
+
+  context "when the dfe_analytics feature is enabled" do
+    before { FeatureFlag.activate(:dfe_analytics) }
+
+    let(:request_uuid) { SecureRandom.uuid }
+
+    it "saves the request_uuid on the big query event" do
+      cpd_lead_provider = create(:cpd_lead_provider, name: "Ambition")
```

Comment:
Do we need all this setup? it _looks_ like it would still run successfully without an auth token, but I could be wrong